### ISSUE TITLE
NPE in EC2FleetCloud.java:319 breaking everything

### ISSUE THREAD

**Issue by jzila** [jenkinsci/ec2-fleet-plugin#78]
```
Timer task com.amazon.jenkins.ec2fleet.CloudNanny@a7a8e60 failed
java.lang.NullPointerException
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud.updateStatus(EC2FleetCloud.java:319)
	at com.amazon.jenkins.ec2fleet.CloudNanny$1.call(CloudNanny.java:46)
	at com.amazon.jenkins.ec2fleet.CloudNanny$1.call(CloudNanny.java:43)
	at hudson.model.Queue._withLock(Queue.java:1438)
	at hudson.model.Queue.withLock(Queue.java:1299)
	at com.amazon.jenkins.ec2fleet.CloudNanny.doRun(CloudNanny.java:43)
	at hudson.triggers.SafeTimerTask.run(SafeTimerTask.java:72)
	at jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

```
May 29, 2019 5:34:05 PM INFO com.amazon.jenkins.ec2fleet.EC2FleetCloud provisionInternal
Start provision label = linux, excessWorkload = 4
May 29, 2019 5:34:05 PM WARNING com.amazon.jenkins.ec2fleet.EC2FleetCloud provision
provisionInternal failed
java.lang.NullPointerException
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud.updateStatus(EC2FleetCloud.java:319)
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud.provisionInternal(EC2FleetCloud.java:260)
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud$1.call(EC2FleetCloud.java:247)
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud$1.call(EC2FleetCloud.java:244)
	at hudson.model.Queue._withLock(Queue.java:1438)
	at hudson.model.Queue.withLock(Queue.java:1299)
	at com.amazon.jenkins.ec2fleet.EC2FleetCloud.provision(EC2FleetCloud.java:244)
	at hudson.slaves.NodeProvisioner$StandardStrategyImpl.apply(NodeProvisioner.java:714)
	at hudson.slaves.NodeProvisioner.update(NodeProvisioner.java:320)
	at hudson.slaves.NodeProvisioner.access$000(NodeProvisioner.java:62)
	at hudson.slaves.NodeProvisioner$NodeProvisionerInvoker.doRun(NodeProvisioner.java:808)
	at hudson.triggers.SafeTimerTask.run(SafeTimerTask.java:72)
	at jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:58)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jzila** [comment]
cc @terma

**Comment by terma** [comment]
@jzila thx, that bug in last release, because of ```private transient EC2Api ec2Api = new EC2Api();``` which will not be restored during jenkins load, fixing.

**Comment by terma** [comment]
@jzila thx for report, fixed in [1.3.1](https://github.com/jenkinsci/ec2-fleet-plugin/releases/tag/ec2-fleet-1.3.1) please ping me if you still have problem, I will keep this issue open.

**Comment by jzila** [comment]
Works now, thanks!
